src/ILVerification/tests/TestDataLoader.cs

```diff
@@ -32,6 +32,78 @@ class TestDataLoader
 
         private const string SpecialTestPrefix = "special.";
 
+        /// <summary>
+        ///  Returns all class correctly implement based on following naming convention
+        ///  [FriendlyName]_ValidTypeImplementation_Valid
+        /// </summary>
+        /// <returns></returns>
+        public static TheoryData<TestCase> GetTypesWithValidTypeImplementation()
+        {
+            var typeSelector = new Func<string[], TypeDefinitionHandle, TestCase>((mparams, typeDefinitionHandle) =>
+            {
+                if (mparams[1] == "ValidTypeImplementation")
```

Comment:
I would just call this `ValidType` and `InvalidType`. Skip implementation

Reply:
fixed